ecommerce-ui/src/components/ShoppingCart.js

```diff
@@ -0,0 +1,61 @@
+
+import React from 'react';
+
+export default function ShoppingCart(props) {
```

Comment:
`propTypes` are needed for this component and any other component that has props